Control/Monad/Logger.hs

```diff
@@ -34,7 +34,7 @@ module Control.Monad.Logger
       MonadLogger(..)
     , MonadLoggerIO (..)
     , LogLevel(..)
-    , LogSource
+    , LogSource(..)
```

Comment:
What's the purpose of this change?

Reply:
Sorry, I'm new. I didn't know that pushing to my fork would add this commit to this PR. The reason why I did this was that in my local haddocks, there was

```haskell
type LogSource = Text
```

where the `Text` part wasn't hyperlinked. This is what confused me into adding the `(..)`, even though I couldn't find anything about this in the haddock documentation or the Haskell Report. But now I think that simply recompilation fixed it, and I can't reproduce the non-hyperlinked `Text` either. So all in all, this is just plain wrong.